telemetry/otel-collector-config.yaml

```diff
@@ -18,27 +35,39 @@ exporters:
       num_consumers: 10
       queue_size: 10000
       # this should work out to about 8M batches/second for 10s buffer
+  # Data sources: traces, metrics, logs
   logging:
     loglevel: debug
+  # Data sources: metrics
   prometheus:
-    endpoint: "0.0.0.0:1234"
+    endpoint: otel-collector:2113
+    namespace: default
```

Comment:
Please omit this, because it prefixes every metric with `default_`, and this breaks our Grafana dashboard metrics.

Reply:
Ah! I mis-read the docs for this exporter, I thought the `namespace` was mandatory, but it's not.

Removed and confirmed Grafana still works:

<img width="1320" alt="image" src="https://user-images.githubusercontent.com/5374887/223496334-69ea174a-35aa-4dbf-af2a-b868a37d6e13.png">